html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      HTML Forms
+    </title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div class="email">
+        <label for="email">Email</label>
+        <input type="email" name="email" id="email" required>
+      </div>
+      <div class="dropbox">
+        <p>Why is coding cool?</p>
+        <select name="coding" id="coding">
+          <option value="select">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div class="radio">
+        <p>Favorite Language (choose one)</p>
+        <input type="radio" id="html" name="coding_language" value="html" checked="checked">
+        <label for="html">HTML</label>
+        <input type="radio" id="css" name="coding_language" value="css">
+        <label for="css">CSS</label>
+        <input type="radio" id="javaScript" name="coding_language" value="javascript">
+        <label for="javaScript">JavaScript</label>
+      </div>
+      <div class="more_info">
+        <p>Would you like to sign up for more information?</p>
+        <input type="checkbox" id="sign_up" name="sign_up" value="sign_up">
+        <label for="sign_up">Sign me up!</label>
+      </div>
+      <div class="comments">
+        <p>Comments</p>
```

Comment:
This should be a `label` element.

Reply:
![Screenshot (8)](https://user-images.githubusercontent.com/103544497/195173799-a7e89260-7d73-4192-9bcd-d8415497fcd2.png)